### PR TITLE
CLDR-14143 clarify license

### PR DESCRIPTION
**Pull request by srl295**
https://unicode-org.atlassian.net/browse/CLDR-14143

copied LICENSE exactly from CLDR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +68,17 @@ Installation using [bower](http://bower.io):
 CLDR does NOT use Github's issue tracking system to track bugs.  If you find an error in
 the data contained here, please file a new ticket at [Unicode Jira](https://unicode-org.atlassian.net/projects/CLDR/issues)
 
+### Licenses
+
+- Usage of CLDR data and software is governed by the [Unicode Terms of Use](http://www.unicode.org/copyright.html)
+a copy of which is included as [LICENSE](./LICENSE).
```
Comment (tstibbs) [inline README.md:74]: It looks from this PR as if `./LICENSE` actually contains a copy of http://www.unicode.org/license.html, not copyright.html?

Comment (srl295) [reply README.md:74]: copyright.html has a link to license, and explains about the copyright on the data as well. So I'd rather link to copyright.html

Comment (srl295) [reply README.md:74]: in the license it says "Distributed under the Terms of Use in https://www.unicode.org/copyright.html. "

**Review by ayien45 — APPROVED** [review]

**Comment by srl295** [comment]
@JCEmmons going to merge this… can fix if needed

**Comment by srl295** [comment]
Landed in 8bb58f8ae3511e9e52d8c1bd874dd1930d5ea31e

**Pull request merged**